<!DOCTYPE HTML>
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

  <title>Ashok K. Sharma</title>
  
  <meta name="author" content="Ashok K. Sharma">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  <link rel="stylesheet" type="text/css" href="stylesheet.css">
	<link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>üåê</text></svg>">

</head>

<body>
  <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    <tr style="padding:0px">
      <td style="padding:0px">
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr style="padding:0px">
            <td style="padding:2.5%;width:63%;vertical-align:middle">
              <p style="text-align:center">
                <name>Ashok K. Sharma</name>
              </p>
	      <p>
		I am currently a Bioinformatics Scientist at <a href="https://www.cedars-sinai.org">Cedars Sinai Medical Center</a>, where I focus on analyzing high-dimensional multi-omics datasets and imaging mass cytometry to study host-microbiome interactions and their role in disease mechanisms. In addition, I work on collaborative projects addressing key biological questions related to inflammatory bowel diseases, microbial co-localization, and gene expression regulation in the gastrointestinal tract.
              </p>
              <p>
	       Previously, I worked as Scientist-II, Computational Biology, with the Global Drug Safety Research Evaluation (DSRE) team at <a href= "https://www.takeda.com" >Takeda Pharmaceuticals </a>, San Diego. I am collaborating with multidisciplinary teams to expand our predictive and retrospective safety capabilities. My work included leveraging AI/ML approaches for biomarker discovery, developing splicing analysis pipelines for oligonucleotide projects, and enhancing transcriptomic data analysis pipelines to support drug discovery.
              </p>
              <p>
		During my postdoctoral training at <a href="https://www.cedars-sinai.org">Cedars Sinai Medical Center</a>, I worked on <a href=" https://www.gastrojournal.org/article/S0016-5085(22)60660-7/pdf "> SARS-COV-2 Persistence in the GI Tract </a> | <a href=" https://www.gastrojournal.org/article/S0016-5085(22)60575-4/pdf?referrer=https%3A%2F%2Fwww.researchgate.net%2F "> Imaging Mass Cytometry Datasets for Bacterial Co-Localization in Crohn's Disease's </a>, <a href=" https://www.frontiersin.org/articles/10.3389/fimmu.2022.907529/full ">Impact of Human Milk Oligosaccharides Gene Expression and Immune Response in Small and Large Intestine</a>, <a href="data/IBRI_presentation.pdf"> Quantification of Bacterial Growth Rates (PTRs) from metagenomic datasets</a> (*Ongoing). Before joining Cedars, I also worked as a Postdoctoral Associate at the <a href=" https://twin-cities.umn.edu "> University of Minnesota Twin Cities</a>, where I worked on various projects to understand microbiome-host interactions in animal and human models under diverse physiological conditions. Which includes <a href="https://journals.asm.org/doi/abs/10.1128/mSphere.00271-19 "> Primates Gut Bacteriome </a>, <a href="https://www.nature.com/articles/s41522-022-00274-3"> Primates Gut Mycobiome </a>, <a href="https://journals.asm.org/doi/abs/10.1128/msystems.00815-20"> Gut microbial functional adaptations </a>, Oral microbiome in (<a href="https://aacrjournals.org/cancerpreventionresearch/article/13/5/463/47400/Oral-Microbiome-Profiling-in-Smokers-with-and"> oral/head and neck cancer (HNSCC) </a>, and <a href="https://www.tandfonline.com/doi/abs/10.1080/20002297.2022.2123624"> ex-vivo dental caries</a>), and Animal microbiome of (<a href="https://link.springer.com/article/10.1186/s42523-022-00175-y"> Beef cattle </a>, and <a href="https://www.sciencedirect.com/science/article/abs/pii/S073708062030397X"> Horse</a>).
              </p>
	       <p>
		I did my PhD in Computaional Biology from <a href=" http://metagenomics.iiserb.ac.in"> Metagenomics and Systems Biology Laboratory </a> at <a href=" https://www.iiserb.ac.in"> IISER Bhopal </a> and funded by the <a href=" https://socialprotection.org/connect/stakeholders/india-ministry-human-resource-development-mhrd ">MHRD</a>. 
	       </p> 
		
		<p align=center>
			<a href="data/AshokSharma-CV.pdf">CV</a> / 
			<a href="data/AshokSharma-research.pdf">Research </a> / 
			<a href="mailto:ashoks773@gmail.com">Email</a> / 
			<a href="https://scholar.google.com/citations?user=zm75iYMAAAAJ&hl=en">Google Scholar</a> / 
			<a href="https://twitter.com/ashoks773">Twitter</a> / <a href="https://github.com/ashoks773">Github</a> /

		<!-- Dropdown for Tutorials -->
    			<div class="dropdown">
        			<a href="#">Tutorials <strong> (Under Prep) </strong> </a>
        			<div class="dropdown-content">
				<a href="https://github.com/ashoks773/SingleCell_CrohnsDataAnalysis">1. scRNA-seq tutorial</a> |
				<a href="https://github.com/ashoks773/SpatialTranscriptomicsWorkflow">2. Spatial transcriptomics tutorial</a>
            			 	<!-- Add more tutorials as needed -->
        			</div>
    			</div>
		 </p>   
	   </td>
            <td style="padding:2.5%;width:40%;max-width:40%">
              <a href="images/AshokSharma.jpg"><img style="width:100%;max-width:100%" alt="profile photo" src="images/AshokSharma_circle.jpg" class="hoverZoomLink"></a>
            </td>
          </tr>
	</tbody></table>
	      
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
            <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading>Publications</heading>
              <p>
		Most of my research has focused on developing and implementing advanced computational approaches including AI/ML models to uncover the microbiome's role in the overall growth, health, and well-being of both animals and humans. I have extensively studied diet-microbiome-host interactions using multi-omics data. Below is a list of my representative papers.     </p>
            </td>
          </tr>
	</tbody></table>
	      
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        
	<tr>
                                <td style="padding:20px;width:25%;vertical-align:middle">
                                    <img src="images/seizure.jpg" alt="PontTuset" width="160"
                                        style="border-style: none">
                                </td>
                                <td width="75%" valign="middle">
                                    <a href="https://academic.oup.com/toxsci/advance-article/doi/10.1093/toxsci/kfae148/7921427">
                                        <papertitle>In vitro human ion channel assays predictive of drug-induced seizure</papertitle>
                                    </a>
                                    <br>
                                    <a href="https://www.linkedin.com/in/jennifer-cohen-phd-dabt-57740914/">Jennifer D Cohen</a>,
                                    <a href="https://www.linkedin.com/in/daheayou/">Dahea You</a>,
                                    <strong>Ashok K. Sharma</strong>,
                                    <a href="https://www.linkedin.com/in/takafumi-takai-a3aa6a128/">Takafumi Takai</a>,
                                    <a href="https://www.linkedin.com/in/hideto-hara/">Hideto Hara</a>,
                                    <a href="https://www.linkedin.com/in/vicencia-toledo-sales/">Vicencia T Sales</a>,
                                    Li Wang,
                                    <a href="https://www.linkedin.com/in/tomoya-yukawa-452055125/">Tomoya Yukawa</a>,
                                    <a href="https://www.linkedin.com/in/beibeicai/">Beibei Cai</a>,
                                    <br>
                                    <em>Toxicological Sciences</em>, 2024
                                    <br>
                                    <a
                                        href="https://academic.oup.com/toxsci/advance-article/doi/10.1093/toxsci/kfae148/7921427">paper</a>
                                    <p></p>
                                    <p>
					We evaluated an in vitro panel of 11 ion channel targets linked to seizure risk using automated electrophysiology, identifying Cav2.1 as the most predictive target for seizurogenic activity. The study demonstrates the value of expanding target panels to improve early prediction of seizure risks in drug discovery.                              
				</td>
                            </tr>
	
	<tr>
                                <td style="padding:20px;width:25%;vertical-align:middle">
                                    <img src="images/wilson_disease.jpg" alt="PontTuset" width="160"
                                        style="border-style: none">
                                </td>
                                <td width="75%" valign="middle">
                                    <a href="https://journals.lww.com/hepcomm/fulltext/2023/10010/the_role_of_the_intestine_in_metabolic.1.aspx">
                                        <papertitle>The role of the intestine in metabolic dysregulation in murine Wilson disease</papertitle>
                                    </a>
                                    <br>
                                    <a href="https://www.linkedin.com/in/gaurav-sarode-ba860740">Gaurav V Sarode</a>,
                                    <a href="https://www.linkedin.com/in/tagreed-mazi/">Tagreed Mazi</a>,
                                    <a href="https://www.linkedin.com/in/kari-neier/">Kari Neier</a>,
                                    Shibata, Noreene M,
                                    <a href="https://microbiome.ucdavis.edu/people/guillaume-jospin">Guillaume Jospin </a>,
                                    Nathaniel Harder  H.O,
                                    AmandaCaceres, 
                                    <a href="https://chemistry.ucdavis.edu/people/marie-heffern">Marie Heffern </a>,
                                    <strong>Ashok K. Sharma</strong>,
                                    <a href="https://scholar.google.com/citations?user=rqTy1G8AAAAJ&hl=en">Shyam More</a>,
                                    <a href="https://profiles.ucdavis.edu/maneesh.dave">Maneesh Dave</a>,
                                    <a href="https://health.usnews.com/doctors/shannon-schroeder-1824545">Shannon Schroeder</a>,
                                    Li Wang,
                                    <a href="https://health.ucdavis.edu/medmicro/faculty/lasalle/">Janine LaSalle</a>,
                                    <a href="https://www.hopkinsmedicine.org/research/labs/svetlana-lutsenko-laboratory">Svetlana Lutsenko</a>,
                                    <a href="https://health.ucdavis.edu/medical-center/team/21912/valentina-medici---gastroenterology-and-hepatology-sacramento/">Valentina Medici</a>
                                    <br>
                                    <em>Hepatology Communications</em>, 2023
                                    <br>
                                    <a
                                        href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10497250/">paper</a>
                                    <p></p>
                                    <p>
                                        Intestine-specific ATP7B deficiency affected both intestinal and systemic response to a high-fat challenge but not the microbiome profile, at least at early stages.
                                    </p>
                                </td>
                            </tr>
		
	<tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/ff.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.cambridge.org/core/journals/gut-microbiome/article/regular-consumption-of-lactofermented-vegetables-has-greater-effects-on-the-gut-metabolome-compared-with-the-microbiome/A6692CAC12EC517CA9545DFDE577C1D4">
                <papertitle>Regular consumption of lacto-fermented vegetables has greater effects on the gut metabolome compared with the microbiome</papertitle> 
	      </a>
              <br>
              <a href="https://www.linkedin.com/in/kylene-guse-2b4135197/">Kylene Guse</a>,
	      <strong>Ashok K. Sharma</strong>,
              <a href="https://www.linkedin.com/in/weyenberg/">Emily Weyenberg</a>,
	      <a href="https://www.linkedin.com/in/samuel-davison-b3a93ba9/">Samuel Davison</a>,
	      <a href="https://www.linkedin.com/in/yiwei-ma/">Yiwei Ma</a>,
	      <a href="https://directory.sph.umn.edu/bio/sph-a-z/abigail-johnson">Abigail Johnson</a>,
	      <a href="https://fscn.cfans.umn.edu/people/chi-chen">Chi Chen</a>,      
	      <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>
              <br>
              <em>Gut Microbiome</em>, 2023
              <br>
              <a href="https://www.cambridge.org/core/journals/gut-microbiome/article/regular-consumption-of-lactofermented-vegetables-has-greater-effects-on-the-gut-metabolome-compared-with-the-microbiome/A6692CAC12EC517CA9545DFDE577C1D4">paper</a>
              <p></p>
              <p>
              We investigated the impact of lacto-fermented vegetable (LFV) consumption on the gut microbiome and metabolome by comparing faecal microbial diversity, composition, and metabolite profiles between LFV consumers and non-consumers. Results revealed minor microbiome changes but significant alterations in the faecal metabolome, including increased abundances of butyrate, acetate, and valerate, highlighting the need for further research on LFV's health benefits.
            </td>
          </tr>	
		
<tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/foodmicrobiome.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://pubs.rsc.org/en/content/articlelanding/2023/fo/d3fo03438k/unauth">
                <papertitle>Effect of red wheat, aleurone, and testa layers on colon cancer biomarkers, nitrosative stress, and gut microbiome composition in rats</papertitle> 
	      </a>
              <br>
              <a href="https://www.linkedin.com/in/chelsey-fiecke-978932179/">Chelsey Fiecke</a>,
	      <a href="https://ag.purdue.edu/directory/ssimsek">Senay Simsek</a>,
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://fscn.cfans.umn.edu/people/daniel-d-gallaher">Daniel D Gallaher</a>,
              <br>
              <em>Food & Function</em>, 2023
              <br>
              <a href="https://pubs.rsc.org/en/content/articlelanding/2023/fo/d3fo03438k/unauth">paper</a>
              <p></p>
              <p>
		We investigated whether the phenolic-rich aleurone and testa layers of red wheat contribute to its chemopreventive effects against colon cancer in a rat model. Red wheat diets reduced biomarkers of cancer risk and altered gut microbiota composition, including increased diversity and changes in key taxa, suggesting the potential roles of dietary fiber and phenolics in reducing colon cancer risk.            </td>
          </tr>	
		
		<tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/dental_caries.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.tandfonline.com/doi/full/10.1080/20002297.2022.2123624">
                <papertitle>Functional signatures of ex-vivo dental caries onset</papertitle> 
	      </a>
              <br>
              <a href="https://www.apariciolab.com/dina-moussa">Dina Moussa</a>,
	      <strong>Ashok K. Sharma</strong>,
              <a href="https://gwdocs.com/profile/tamer-mansour">Tamer Mansour</a>,
	      <a href="https://cbs.umn.edu/contacts/bruce-witthuhn">Bruce Witthuhn</a>,
	      <a href="https://dentistry.umn.edu/faculty/jorge-perdigao-henriques">Jorge Perdigao Henriques</a>,
	      <a href="https://guide.americananthro.org/6030/Joel-Rudney">Joel Rudney</a>,
	      <a href="https://dentistry.umn.edu/faculty/conrado-aparicio-badenas">Conrado Aparicio Badenas</a>,
	      <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>
              <br>
              <em>Journal of Oral Microbiology</em>, 2022
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/sra/PRJNA668810">raw data</a> /
              <a href="https://www.tandfonline.com/doi/full/10.1080/20002297.2022.2123624">paper</a>
              <p></p>
              <p>
              We optimize ex-vivo model integrated with the advanced imaging combined with multi-omics libraries to detect functional signatures associated with dental caries.
              </p>
            </td>
          </tr>	
		
           <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/covid_micro.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9212536/">
                <papertitle>In-Patient Antibiotic Exposure Promotes SARS-COV-2 Persistence in the Gi Tract in Covid-19 Admitted Patients</papertitle> 
	      </a>
              <br>
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://www.linkedin.com/in/anthony-martin-19962189/">Anthony Martin</a>,
              <a href="https://scholar.google.com/citations?user=mXfLnYwAAAAJ&hl=en">Jacob Moskowitz</a>,
	      <a href="https://www.linkedin.com/in/stephanie-bora-1343a089/">Stephanie Bora</a>,
	      <a href="https://www.linkedin.com/in/katherinelagree/">Katherine Lagree</a>,
	      <a href="https://dorresteinlab.ucsd.edu">Pieter Dorrestein</a>,
	      <a href="https://www.cedars-sinai.edu/research/labs/underhill.html">David Underhill</a>,
	      <a href="https://knightlab.ucsd.edu">Rob Knight</a>,
	      <a href="https://www.cedars-sinai.org/provider/peter-chen-1272861.html">Peter Chen</a>,
	      <a href="http://www.devkotalab.com">Suzanne Devkota</a>
              <br>
              <em>Gastroenterology</em>, 2022 &nbsp <font color="red"><strong>(Oral Presentation)</strong></font>
              <br>
	      <a href="https://qiita.ucsd.edu/study/description/13430">raw data</a> /
	      <a href="data/DDW_Ashok_2022.pdf">presentation</a> /     
              <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9212536/pdf/main.pdf">abstract</a>
              <p></p>
              <p>
		The heavy-dose antibiotic regimen administered to COVID-19 in-patients is associated with viral persistence of SARS-COV-2 in the GI tract, suggesting an important role of the gut microbiome in excluding SARS-COV-2 from the GI tract, perhaps by competitive exclusion or promoting interferon responses.              </p>
            </td>
          </tr>	 
		
           <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/HMO.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.frontiersin.org/articles/10.3389/fimmu.2022.907529/full">
                <papertitle>Human Milk Oligosaccharides Impact Cellular and Inflammatory Gene Expression and Immune Response</papertitle> 
	      </a>
              <br>
              <a href="https://www.depts.ttu.edu/vetschool/about/meet-our-team/faculty/fernanda-rosa/">Fernanda Rosa</a>,
	      <strong>Ashok K. Sharma</strong>,
              <a href="https://scholar.google.com/citations?hl=en&user=cyi8za0AAAAJ&view_op=list_works&sortby=pubdate">Manoj Gurung</a>,
	      <a href="https://www.cedars-sinai.edu/research/labs/casero-lab.html">David Casero</a>,
	      <a href="https://loop.frontiersin.org/people/1219452/overview">Katelin Matazel</a>,
	      <a href="http://www.bodelab.com">Lars Bode</a>,
	      <a href="https://www.linkedin.com/in/christy-simecka-822537162/">Christy Simecka</a>,
	      <a href="https://www.asas.org/about/national-awards/past-awardees/award-winner-detail/ahmed-elolimy">Ahmed Elolimy</a>,
	      <a href="https://www.researchgate.net/scientific-contributions/Patricia-Tripp-2217621360">Patricia Tripp</a>,
	      <a href="https://loop.frontiersin.org/people/965579/overview">Christopher E Randolph</a>,
	      <a href="https://www.immunology.pitt.edu/person/timothy-hand-phd">Timothy W Hand</a>,
	      <a href="https://biostatistics.uams.edu/faculty-and-staff/d-keith-williams/">D Keith Williams</a>, 
              <a href="https://vetmed.vt.edu/people/faculty/leroith-tanya.html">Tanya LeRoith</a>,
	      <a href="https://www.ars.usda.gov/people-locations/person/?person-id=53056">Laxmi Yeruva</a>
              <br>
              <em>Frontiers in Microbiology </em>, 2022
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/bioproject/814680">raw data</a> /
              <a href="https://www.frontiersin.org/articles/10.3389/fimmu.2022.907529/full">paper</a> /
	      <a href="https://github.com/ashoks773/Liner_mixed_model_RNAseq">project page</a>
              <p></p>
              <p>
		In this study, we have shown that HMO administration appears to promote immunoregulatory effects at the gene expression level in the absence of gut microbiota.
	      </p>
            </td>
          </tr>	   

		<tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/Dietary_zinc.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.cambridge.org/core/journals/journal-of-nutritional-science/article/dietary-zinc-restriction-affects-the-expression-of-genes-related-to-immunity-and-stress-response-in-the-small-intestine-of-pigs/657F393613FF6797305C92CE8BEF6B0B">
                <papertitle>Dietary zinc restriction affects the expression of genes related to immunity and stress response in the small intestine of pigs</papertitle> 
	      </a>
              <br>
	      <a href="https://www.linkedin.com/in/ramya-lekha-medida-a09875160/">Ramya Lekha Medida</a>,
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://www.linkedin.com/in/yue-guo-5933761bb/">Yue Guo</a>,
              <a href="https://ansci.umn.edu/people/lee-johnston">Lee Johnston</a>,
	      <a href="https://ansci.umn.edu/people/pedro-urriola">Pedro Urriola</a>,
	      <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>,
	      <a href="https://ansci.umn.edu/people/milena-saqui-salces">Milena Saqui Salces</a>
              <br>
              <em>Journal of Nutritional Science</em>, 2022
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE181343">raw data</a> /
	      <a href="https://www.cambridge.org/core/journals/journal-of-nutritional-science/article/dietary-zinc-restriction-affects-the-expression-of-genes-related-to-immunity-and-stress-response-in-the-small-intestine-of-pigs/657F393613FF6797305C92CE8BEF6B0B">paper</a>   
              <p></p>
              <p>
		The main goal of the present study was to understand intestinal responses to nutritional Zn restriction and supplementation that could influence overall health of the animal and indicate early changes in response to Zn restriction. </p>
            </td>
          </tr>	 
		
	    <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/Tobacco_oralMicro.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.frontiersin.org/articles/10.3389/fcimb.2022.841465/full">
                <papertitle>Human Assessing the Effect of Smokeless Tobacco Consumption on Oral Microbiome in Healthy and Oral Cancer Patients</papertitle> 
	      </a>
              <br>
              <a href="https://scholar.google.co.in/citations?user=JLeu3F4AAAAJ&hl=en">Rituja Saxena</a>,
	      <a href="https://www.linkedin.com/in/vishnu-prasoodanan-p-k-ph-d-7a672072/?originalSubdomain=in">Vishnu Prasoodanan</a>,
	      <a href="https://loop.frontiersin.org/people/1609862/overview">Sonia Vidushi Gupta</a>,
	      <a href="https://loop.frontiersin.org/people/254248/overview">Sudheer Gupta</a>,
	      <a href="https://loop.frontiersin.org/people/1609654/overview">Prashant Waiker</a>,
	      <a href="https://loop.frontiersin.org/people/1465219/overview">Atul Samaiya</a>,
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a>
              <br>
              <em>Frontiers in Cellular and Infection Microbiology </em>, 2022
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA789915">raw data</a> /
              <a href="https://www.frontiersin.org/articles/10.3389/fcimb.2022.841465/full">paper</a>
              <p></p>
              <p>
		In this study, we profiled oral microbiome of healthy and oral cancer patients to decipher the microbial dysbiosis due to the consumption of smokeless-tobacco-based products and also revealed the tobacco-associated microbiome. 	      </p>
            </td>
          </tr>	
		
	<tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/BeefCattle_micro.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://link.springer.com/article/10.1186/s42523-022-00175-y">
                <papertitle>Microbiome network traits in the rumen predict average daily gain in beef cattle under different backgrounding systems</papertitle> 
	      </a>
              <br>
              <a href="https://scholar.google.com/citations?user=a2UnfJ4AAAAJ&hl=en">Bobwealth Oakina Omontese</a>,
	      <strong>Ashok K. Sharma</strong>,
              <a href="https://www.linkedin.com/in/samuel-davison-b3a93ba9/">Samuel Davison</a>,
	      <a href="https://www.linkedin.com/in/emily-jacobson-b0b49b23a/">Emily Jacobson</a>,
	      <a href="https://mnnutritionconf.umn.edu/people/alfredo-dicostanzo-phd">Alfredo DiCostanzo</a>,
	      <a href="https://www.linkedin.com/in/megan-webb-ph-d-08b730a3/">Megan Webb</a>,
	      <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>
              <br>
              <em>Animal Microbiome </em>, 2022
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA763290">raw data</a> /
              <a href="https://link.springer.com/article/10.1186/s42523-022-00175-y">paper</a>
              <p></p>
              <p>
		This longitudinal study was designed to assess how BKG affects rumen bacterial communities and average daily gain (ADG) in beef cattle.            </td>
          </tr>	
		
	  <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/primates_mycobiome.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.nature.com/articles/s41522-022-00274-3">
                <papertitle>The primate gut mycobiome-bacteriome interface is impacted by environmental and subsistence factors</papertitle> 
	      </a>
              <br>
	      <strong>Ashok K. Sharma</strong>,
              <a href="https://www.linkedin.com/in/samuel-davison-b3a93ba9/">Samuel Davison</a>,
	      <a href="https://www.ivb.cz/en/person/pafco-barbora/">Barbora Pafco</a>,
	      <a href="https://foodforhealth.unl.edu/jonathan-b-clayton">Jonathan B. Clayton</a>,
	      <a href="https://hunter.cuny.edu/people/jessica-rothman/">Jessica Rothman</a>,
	      <a href="https://www.brookes.ac.uk/templates/pages/staff.aspx?wid=&op=full&uid=p0026444">Matthew McLennan</a>,      
	      <a href="https://www.semanticscholar.org/author/Marie-Cibot/6539438">Marie Cibot</a>,
	      <a href="https://www.linkedin.com/in/terence-fuh-09637159/?originalSubdomain=cf">Terence Fuh</a>,
              <a href="https://www.researchgate.net/scientific-contributions/Roman-Vodicka-2054683289">Roman Vodicka</a>,
	      <a href="https://uncw.academia.edu/CarolynJostRobinson">Carolyn Jost Robinson</a>,
              <a href="https://www.ivb.cz/en/person/klara-petrzelkova/">Klara Petrzelkova</a>,
	      <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>
              <br>
              <em>npj Biofilms and Microbiomes </em>, 2022
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA686661">raw data</a> /
	      <a href="https://www.nature.com/articles/s41522-022-00274-3">paper</a> /
              <a href="https://github.com/ashoks773/mycobiome_bacteriome">project page </a>
              <p></p>
              <p>
		Our main findings indicate a strong influence of host-ecological factors including diet and lifestyle in shaping the fungal community composition in the primate‚Äôs gut. This is in contrast with the gut bacterial fraction, which seems to be more influenced by the host genetics.           
	     </tr>	
		
	<tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/Indian_oralMicrobiome.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.nature.com/articles/s41522-021-00248-x">
                <papertitle>Western and non-western gut microbiomes reveal new roles of Prevotella in carbohydrate metabolism and mouth‚Äìgut axis</papertitle> 
	      </a>
              <br>
	      <a href="https://www.linkedin.com/in/vishnu-prasoodanan-p-k-ph-d-7a672072/?originalSubdomain=in">Vishnu Prasoodanan</a>,
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://www.researchgate.net/profile/Shruti-Mahajan-4">Shruti Mahajan</a>,
	      <a href="https://scholar.google.com/citations?user=Gcp89foAAAAJ&hl=en">Darshan Dhakan</a>,
              <a href="https://www.scarialab.com">Joy Scaria</a>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a>
              <br>
              <br>
              <em>npj biofilms and microbiomes </em>, 2021
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA397112">raw data</a> /
	      <a href="https://www.nature.com/articles/s41522-021-00248-x">paper</a>  
              <p></p>
              <p>
		A population-wide analysis was carried out on 586 healthy samples from western and non-western populations including the largest Indian cohort comprising of 200 samples, and 189 Inflammatory Bowel Disease samples from western populations.	
	</tr> 
        
        <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/primates_shotgunMetagenome.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://journals.asm.org/doi/full/10.1128/mSystems.00815-20">
                <papertitle>Traditional Human Populations and Nonhuman Primates Show Parallel Gut Microbiome Adaptations to Analogous Ecological Conditions</papertitle> 
	      </a>
              <br>
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://www.ivb.cz/en/person/klara-petrzelkova/">Klara Petrzelkova</a>,
	      <a href="https://www.ivb.cz/en/person/pafco-barbora/">Barbora Pafco</a>,
	      <a href="https://uncw.academia.edu/CarolynJostRobinson">Carolyn Jost Robinson</a>,
	      <a href="https://www.linkedin.com/in/terence-fuh-09637159/?originalSubdomain=cf">Terence Fuh</a>,
	      <a href="https://asm.org/Biographies/Brenda-Anne-Wilson,-Ph-D">Brenda Anne Wilson</a>,
	      <a href="https://anthro.illinois.edu/directory/profile/rstumpf">Rebecca M. Stumpf</a>,
	      <a href="https://typeset.io/authors/manolito-torralba-pa9xjudb0o">Manolito Torralba</a>,
	      <a href="http://blekhmanlab.org">Ran Blekhman</a>,
	      <a href="https://nutrsci.illinois.edu/directory/bwhite44">Bryan White</a>,
	      <a href="http://www.nasonline.org/member-directory/members/20013261.html">Karen E. Nelson</a>,
	      <a href="https://www.colorado.edu/anthropology/steven-leigh">Steven Leigh</a>,
              <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>
	     <br>
              <br>
              <em>mSystems </em>, 2020
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA635116">raw data</a> /
	      <a href="https://journals.asm.org/doi/full/10.1128/mSystems.00815-20">paper</a> /
	      <a href="https://github.com/ashoks773/Primates-Gut-Metagenome">project page </a>
              <p></p>
              <p>
                The results of this study highlight parallel gut microbiome traits in human and nonhuman primates, depending on subsistence strategy.
	</tr> 
		
        <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/Horse_microbiome.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.sciencedirect.com/science/article/abs/pii/S073708062030397X">
                <papertitle>The Horse Gut Microbiome Responds in a Highly Individualized Manner to Forage Lignification</papertitle> 
	      </a>
              <br>
	      <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>,
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://agnr.umd.edu/about/directory/amanda-grev/">Amanda Grev</a>,
	      <a href="https://agronomy.cfans.umn.edu/people/craig-c-sheaffer">Craig C Sheaffer</a>,
	      <a href="https://ansci.umn.edu/people/krishona-martinson">Krishona Martison</a>  
	     <br>
              <br>
              <em>Journal of Equine Veterinary Science </em>, 2020
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA668801">raw data</a> /
	      <a href="https://www.sciencedirect.com/science/article/pii/S073708062030397X">paper</a>
              <p></p>
              <p>
		Results in this study indicate that the horse gut microbiome responds in an individualized manner to changes in the amount of acid detergent lignin in alfalfa hay, potentially impacting several feed digestibility characteristics.
	</tr> 
		
        <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/molib.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.sciencedirect.com/science/article/pii/S088875431930549X">
                <papertitle>Molib: A machine learning based classification tool for the prediction of biofilm inhibitory molecules</papertitle> 
	      </a>
              <br>
	      <a href="https://scholar.google.com/citations?user=N_gpB7YAAAAJ&hl=en">Gopal Narayan Srivastava</a>,
              <a href="https://www.linkedin.com/in/aditya-malwe-9a9279176/?originalSubdomain=in">Aditya Malwe</a>,
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://www.linkedin.com/in/vibhuti-shastri-952580126/?originalSubdomain=in">Vibhuti Shastri</a>,
	      <a href="https://www.linkedin.com/in/keshav-hibare-8003b41b1/?originalSubdomain=in">Keshav Hibare</a>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a> 
	     <br>
              <br>
              <em>Genomics </em>, 2020
              <br>
	      <a href="https://www.sciencedirect.com/science/article/pii/S073708062030397X">paper</a> /
	      <a href="http://metagenomics.iiserb.ac.in/Molib/">web page</a>    
              <p></p>
              <p>
		We developed the ‚ÄòMolib‚Äô tool to predict the biofilm inhibitory activity of small molecules.
	</tr> 
	
         <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/headneck_cancer_microbiome.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://aacrjournals.org/cancerpreventionresearch/article/13/5/463/47400/Oral-Microbiome-Profiling-in-Smokers-with-and">
                <papertitle>Oral Microbiome Profiling in Smokers with and without Head and Neck Cancer Reveals Variations Between Health and Disease</papertitle> 
	      </a>
              <br>
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://profiles.ucsf.edu/william.debusk">William Debusk</a>,
              <a href="https://directory.sph.umn.edu/bio/sph-a-z/irina-stepanov">Irina Stepanov</a>,
	      <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>,
	      <a href="https://med.umn.edu/bio/ent-faculty/samir-khariwala">Samir Khariwala</a> 
	     <br>
              <br>
              <em>Cancer Prevention Research </em>, 2020
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/sra/PRJNA668810">raw data</a> /
	      <a href="https://aacrjournals.org/cancerpreventionresearch/article/13/5/463/47400/Oral-Microbiome-Profiling-in-Smokers-with-and">paper</a> /
	      <a href="https://github.com/ashoks773/Oral-microbiome-16S">project page</a>   
              <p></p>
              <p>
	       To determine microbial associations with oral/head and neck cancer (HNSCC) among tobacco users, we characterized oral microbiome composition in smokers with and without HNSCC. 16S rRNA MiSeq sequencing was used to examine the oral mucosa microbiome of 27 smokers with (cases) and 24 without HNSCC (controls). 
	</tr> 
		 
         <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/CRC_microbiome.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://journals.asm.org/doi/full/10.1128/mSystems.00438-19">
                <papertitle>Association of Flavonifractor plautii, a Flavonoid-Degrading Bacterium, with the Gut Microbiome of Colorectal Cancer Patients in India</papertitle> 
	      </a>
              <br>
	      <a href="https://www.linkedin.com/in/ankit-gupta-82a632103/">Ankit Gupta</a>,
	      <a href="https://scholar.google.com/citations?user=Gcp89foAAAAJ&hl=en">Darshan Dhakan</a>,
	      <a href="https://www.linkedin.com/in/abhijit-maji-66861643/">Abhijit Maji</a>,
	      <a href="https://scholar.google.co.in/citations?user=JLeu3F4AAAAJ&hl=en">Rituja Saxena</a>,
	      <a href="https://www.linkedin.com/in/vishnu-prasoodanan-p-k-ph-d-7a672072/?originalSubdomain=in">Vishnu Prasoodanan</a>,
	      <a href="https://www.researchgate.net/profile/Shruti-Mahajan-4">Shruti Mahajan</a>,   
	      <a href="https://www.linkedin.com/in/joby-pulikkan-523b63125/?originalSubdomain=in">Joby Pulikkan</a>,
	      <a href="https://mobile.twitter.com/kjacob125">Kurian Jacob</a>,
	      <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>,
              <a href="https://www.scarialab.com">Joy Scaria</a>,
	      <a href="https://www.kramato.com">Katherine R. Amato</a>,
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a> 
	     <br>
              <br>
              <em>mSystems </em>, 2019
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA531273">raw data</a> /
	      <a href="https://journals.asm.org/doi/full/10.1128/mSystems.00438-19">paper</a> 
              <p></p>
              <p>
		This study provides novel insights on the CRC-associated microbiome of a unique cohort in India, reveals the potential role of a new bacterium in CRC, and identifies cohort-specific biomarkers, which can potentially be used in noninvasive diagnosis of CRC.
		</tr> 
	
         <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/primates_bacteriome.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://journals.asm.org/doi/full/10.1128/mSphere.00271-19">
                <papertitle>Plasticity in the Human Gut Microbiome Defies Evolutionary Constraints</papertitle> 
	      </a>
              <br>
	      
	      <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>,
              <strong>Ashok K. Sharma**</strong>,
              <a href="https://scholar.google.com/citations?user=lfXDRVcAAAAJ&hl=en">Elizabeth K Mallott</a>,
	      <a href="https://www.ivb.cz/en/person/klara-petrzelkova/">Klara Petrzelkova</a>,
	      <a href="https://uncw.academia.edu/CarolynJostRobinson">Carolyn Jost Robinson</a>,
	      <a href="http://animalrange.montana.edu/directory/faculty/1524775/carl-yeoman">Carl J Yeoman</a>,
	      <a href="https://medicine.wsu.edu/overview/faculty-and-staff/franck-carbonero-phd/">Franck Carbonero</a>,
	      <a href="https://www.ivb.cz/en/person/pafco-barbora/">Barbora Pafco</a>,
	      <a href="https://hunter.cuny.edu/people/jessica-rothman">Jessica Rothman</a>,
	      <a href="https://www.researchgate.net/profile/Alexander-Ulanov-5">Alexander Ulanov</a>,
	      <a href="https://www.linkedin.com/in/klara-vlckova-985905197/?originalSubdomain=ie">Klara Vlckova</a>,
	      <a href="https://www.kramato.com">Katherine R. Amato</a>,
	      <a href="https://scholar.google.de/citations?user=bYq0R54AAAAJ&hl=en">Stephanie L. Schnorr</a>,
              <a href="https://anthropology.dartmouth.edu/people/nathaniel-j-dominy">Nathaniel J Dominy</a>,
	      <a href="https://www.sci.muni.cz/en/about-us/faculty-staff/241525-david-modry">David Modry</a>,
	      <a href="https://orcid.org/0000-0002-9207-4925">Angelique Todd</a>,
	      <a href="https://typeset.io/authors/manolito-torralba-pa9xjudb0o">Manolito Torralba</a>,
	      <a href="http://www.nasonline.org/member-directory/members/20013261.html">Karen E. Nelson</a>,
	      <a href="http://www.burns-lab.org">Michael Burns</a>, 
	      <a href="http://blekhmanlab.org">Ran Blekhman</a>,
	      <a href="https://melissaremis.com">Melissa J. Remis</a>,
	      <a href="https://anthro.illinois.edu/directory/profile/rstumpf">Rebecca M. Stumpf</a>,
              <a href="https://asm.org/Biographies/Brenda-Anne-Wilson,-Ph-D">Brenda Anne Wilson</a>,
	      <a href="https://ansc.illinois.edu/directory/hgaskins">Rex Gaskins</a>,  
	      <a href="https://anthro.illinois.edu/directory/profile/p-garber">Paul A Garber</a>,
	      <a href="https://nutrsci.illinois.edu/directory/bwhite44">Bryan White</a>,
	      <a href="https://www.colorado.edu/anthropology/steven-leigh">Steven Leigh</a>
	     <br>
              <br>
              <em>mSphere </em>, 2019
              <br>
	      <a href="https://journals.asm.org/doi/full/10.1128/mSphere.00271-19">paper</a> /
	      <a href="https://github.com/ashoks773/Primates-GutMicrobiome-16S">project page</a>
              <p></p>
              <p>
		The results of this study indicate a discordance between gut microbiome composition and evolutionary history in primates, calling into question previous notions about host genetic control of the primate gut microbiome.
	</tr> 
		 
	<tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/fecal_RNAseq.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-5813-z">
                <papertitle>Mapping gastrointestinal gene expression patterns in wild primates and humans via fecal RNA-seq</papertitle> 
	      </a>
              <br>
              <strong>Ashok K. Sharma</strong>,
	      <a href="https://www.ivb.cz/en/person/pafco-barbora/">Barbora Pafco</a>,
	      <a href="https://www.linkedin.com/in/klara-vlckova-985905197/?originalSubdomain=ie">Klara Vlckova</a>,
              <a href="https://www.researchgate.net/profile/Barbora-Cervena">Barbora Cervena</a>
              <a href="https://web.natur.cuni.cz/zoologie/biodiversity/index.php?page=kreisinger">Jakub Kreisinger</a>,
	      <a href="https://www.linkedin.com/in/samuel-davison-b3a93ba9/">Samuel Davison</a>,
	      <a href="https://www.linkedin.com/in/karen-beeri-b5453911/">Karen Beeri</a>,
	      <a href="https://www.linkedin.com/in/terence-fuh-09637159/?originalSubdomain=cf">Terence Fuh</a>, 
	      <a href="https://www.colorado.edu/anthropology/steven-leigh">Steven Leigh</a>,
	      <a href="http://www.burns-lab.org">Michael Burns</a>,
	      <a href="http://blekhmanlab.org">Ran Blekhman</a>,       
	      <a href="https://www.ivb.cz/en/person/klara-petrzelkova/">Klara Petrzelkova</a>,
	      <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>
	     <br>
              <br>
              <em>BMC Genomics </em>, 2019
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA541574">raw data</a> /
	      <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-5813-z">paper</a>
              <p></p>
              <p>
		Results in this study suggest that fecal RNA-seq, targeting gastrointestinal epithelial cells can be used to evaluate primate intestinal physiology and gut gene regulation, in samples obtained in challenging conditions in situ. 
	</tr> 	 
		
        <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/goirlla_micro_agesex.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ajpa.23842">
                <papertitle>Gut microbiome composition of wild western lowland gorillas is associated with individual age and sex factors</papertitle> 
	      </a>
              <br>
              <a href="https://www.ivb.cz/en/person/pafco-barbora/">Barbora Pafco</a>,
	      <strong>Ashok K. Sharma**</strong>
	      <a href="https://www.ivb.cz/en/person/klara-petrzelkova/">Klara Petrzelkova</a>,
	      <a href="https://www.linkedin.com/in/klara-vlckova-985905197/?originalSubdomain=ie">Klara Vlckova</a>,
	      <a href="https://orcid.org/0000-0002-9207-4925">Angelique Todd</a>,
	      <a href="http://animalrange.montana.edu/directory/faculty/1524775/carl-yeoman">Carl J Yeoman</a>,
	      <a href="https://asm.org/Biographies/Brenda-Anne-Wilson,-Ph-D">Brenda Anne Wilson</a>,   
              <a href="https://anthro.illinois.edu/directory/profile/rstumpf">Rebecca M. Stumpf</a>,
	      <a href="https://nutrsci.illinois.edu/directory/bwhite44">Bryan White</a>,  
	      <a href="http://www.nasonline.org/member-directory/members/20013261.html">Karen E. Nelson</a>,  
	      <a href="https://www.colorado.edu/anthropology/steven-leigh">Steven Leigh</a>,          
	      <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>
	     <br>
              <br>
              <em>American Journal of Biological Anthropology </em>, 2019
              <br>
	      <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ajpa.23842">paper</a>
              <p></p>
              <p>
		Objective of this study was to determine the associations between an individual's age and sex on the diversity and composition of the gut microbiome in wild western lowland gorillas.
	</tr>
		
	<tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/indian_gutmicorbiome.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://academic.oup.com/gigascience/article/8/3/giz004/5304367">
                <papertitle>The unique composition of Indian gut microbiome, gene catalogue, and associated fecal metabolome deciphered using multi-omics approaches</papertitle> 
	      </a>
              <br>
	      <a href="https://scholar.google.com/citations?user=Gcp89foAAAAJ&hl=en">Darshan Dhakan</a>,
	      <a href="https://www.linkedin.com/in/abhijit-maji-66861643/">Abhijit Maji</a>,
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://scholar.google.co.in/citations?user=JLeu3F4AAAAJ&hl=en">Rituja Saxena</a>,
	      <a href="https://www.linkedin.com/in/joby-pulikkan-523b63125/?originalSubdomain=in">Joby Pulikkan</a>,
	      <a href="https://www.researchgate.net/lab/Tony-Grace-Lab">Tony Grace</a>,
	      <a href="https://ansci.umn.edu/people/andres-gomez">Andres Gomez</a>,
              <a href="https://www.scarialab.com">Joy Scaria</a>,
	      <a href="https://www.kramato.com">Katherine R. Amato</a>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a> 
	     <br>
              <br>
              <em>GigaScience </em>, 2019
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA397112">raw data</a> /
	      <a href="https://academic.oup.com/gigascience/article/8/3/giz004/5304367">raw data</a>
              <p></p>
              <p>
		We analyzed the gut microbiome of 110 healthy individuals from two distinct locations (North-Central and Southern) in India using multi-omics approaches, including 16S rRNA gene amplicon sequencing, whole-genome shotgun metagenomic sequencing, and metabolomic profiling of fecal and serum samples.
	</tr>
		
	<tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/pecock_Genome.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.frontiersin.org/articles/10.3389/fgene.2018.00392/full">
                <papertitle>Genome Sequence of Peacock Reveals the Peculiar Case of a Glittering Bird</papertitle> 
	      </a>
              <br>
	      <a href="https://www.linkedin.com/in/shubham-k-jaiswal/?originalSubdomain=in">Subham K. Jaiswal</a>,
	      <a href="https://www.linkedin.com/in/ankit-gupta-82a632103/">Ankit Gupta</a>,
	      <a href="https://scholar.google.co.in/citations?user=JLeu3F4AAAAJ&hl=en">Rituja Saxena</a>,
	      <a href="https://www.linkedin.com/in/vishnu-prasoodanan-p-k-ph-d-7a672072/?originalSubdomain=in">Vishnu Prasoodanan</a>,
	      <strong>Ashok K. Sharma</strong>,      
	      <a href="https://www.linkedin.com/in/parul-mittal-29856b45/">Parul Mittal</a>, 
	      <a href="https://www.linkedin.com/in/ankita-roy-37638592/">Ankita Roy</a>, 
	      <a href="https://loop.frontiersin.org/people/416458/overview"Aaron Shafer</a>,        
	      <a href="https://sites.google.com/view/ceglab/contact?pli=1">Nagarjun Vijay</a>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a> 
	     <br>
              <br>
              <em>Frontiers in Genetics </em>, 2018
              <br>
	      <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRP083005">raw data</a> /
	      <a href="https://www.frontiersin.org/articles/10.3389/fgene.2018.00392/full">paper</a>
              <p></p>
              <p>
		Here, we report the first genome sequence and comparative analysis of peacock with the high quality genomes of chicken, turkey, duck, flycatcher and zebra finch.
	</tr>
		
        <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/digoxin_metabolism.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/jcb.26638">
                <papertitle>Mechanistic and structural insight into promiscuity based metabolism of cardiac drug digoxin by gut microbial enzyme</papertitle> 
	      </a>
              <br>
	      <a href="https://www.linkedin.com/in/kundan-kumar-49635628/">Kundan Kumar</a>,
              <a href="https://www.linkedin.com/in/shubham-k-jaiswal/?originalSubdomain=in">Subham K. Jaiswal</a>,
	      <a href="https://www.linkedin.com/in/gaurao-dhoke-19568629/">Gaurao Dhoke</a>,
	      <a href="https://scholar.google.com/citations?user=N_gpB7YAAAAJ&hl=en">Gopal Narayan Srivastava</a>,
	      <strong>Ashok K. Sharma</strong>,      
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a> 
	     <br>
              <br>
              <em>Journal of Cellular Biochemistry </em>, 2017
              <br>
	      <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/jcb.26638">paper</a>
              <p></p>
              <p>
		We report the atomistic details and energy economics binding and metabolism of Digoxin by the Cgr2 protein of Eggerthella lenta DSM 2243.
	</tr>
		
	 <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/antiinflam.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-016-1103-6">
                <papertitle> Prediction of anti-inflammatory proteins/peptides: an insilico approach</papertitle> 
	      </a>
              <br>
              <a href="https://www.linkedin.com/in/sudheer-gupta-91369580?originalSubdomain=in">Sudheer Gupta</a>,
	      <strong>Ashok K. Sharma**</strong>,
	      <a href="https://www.linkedin.com/in/vibhuti-shastri-952580126/?originalSubdomain=in">Vibhuti Shastri</a>,
	      <a href="https://in.linkedin.com/in/midhun-k-madhu-9a909286">Midhun K. Madhu Dhakan</a>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a>
              <br>
              <br>
              <em>Journal of Translational Medicine </em>, 2017
              <br>
	      <a href="https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-016-1103-6">paper</a> /
	      <a href="http://metagenomics.iiserb.ac.in/antiinflam/">web page </a>
              <p></p>
              <p>
		In this study, we have developed a prediction tool for the classification of peptides as anti-inflammatory epitopes or non anti-inflammatory epitopes.
	</tr> 
			  
	<tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/drugbug.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.nature.com/articles/s41598-017-10203-6">
                <papertitle> Approach for the prediction of species-specific biotransformation of xenobiotic/drug molecules by the human gut microbiota</papertitle> 
	      </a>
              <br>
              <strong>Ashok K. Sharma</strong>,
              <a href="https://www.linkedin.com/in/shubham-k-jaiswal/?originalSubdomain=in">Subham K. Jaiswal</a>
	      <a href="https://www.researchgate.net/profile/Nikhil-Chaudhary-10">Nikhil Chaudhary</a>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a>
              <br>
              <br>
              <em>Scientific Reports </em>, 2017
              <br>
	      <a href="https://www.nature.com/articles/s41598-017-10203-6">paper</a> /
	      <a href="http://metagenomics.iiserb.ac.in/drugbug">web page </a>
              <p></p>
              <p>
		DrugBug is a computational approach to predict the metabolic enzymes and gut bacterial species, which can potentially carry out the biotransformation of a xenobiotic/drug molecule
	</tr> 
		  
		 
         <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/toxim.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.frontiersin.org/articles/10.3389/fphar.2017.00880/full">
                <papertitle>ToxiM: A Toxicity Prediction Tool for Small Molecules Developed Using Machine Learning and Chemoinformatics Approaches</papertitle> 
	      </a>
              <br>
              <strong>Ashok K. Sharma</strong>,
	      <a href="https://scholar.google.com/citations?user=N_gpB7YAAAAJ&hl=en">Gopal Narayan Srivastava</a>,
              <a href="https://www.linkedin.com/in/ankita-roy-37638592/">Ankita Roy</a>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a>
              <br>
              <br>
              <em>Frontiers in Pharmacology </em>, 2017
              <br>
	      <a href="https://www.frontiersin.org/articles/10.3389/fphar.2017.00880/full">paper</a> /
	      <a href="http://metagenomics.iiserb.ac.in/ToxiM/">web page </a>
              <p></p>
              <p>
			ToxiM is a machine-learning based binary classifier for the prediction of toxicity of molecules. Regression models for CaCo-2 permeability and Aqueous solubility has been provided to support the predicted toxicity for a given molecule. For the batch processing please download standalone version of ToxiM from Dataset page.	
	</tr>  
	
		<tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/Biofin.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2016.00949/full">
                <papertitle>Prediction of Biofilm Inhibiting Peptides: An In silico Approach</papertitle> 
	      </a>
              <br>
	      <a href="https://www.linkedin.com/in/sudheer-gupta-91369580?originalSubdomain=in">Sudheer Gupta</a>,
              <strong>Ashok K. Sharma</strong>,
              <a href="https://www.linkedin.com/in/shubham-k-jaiswal/?originalSubdomain=in">Subham K. Jaiswal</a>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a>
              <br>
              <br>
              <em>Frontiers in Microbiology </em>, 2016
              <br>
	      <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2016.00949/full">paper</a> /
	      <a href="http://metagenomics.iiserb.ac.in/biofin/">web page </a>
              <p></p>
              <p>
                  BioFin is a unique computational method for the prediction of biofilm inhibiting peptides. The experimentally validated biofilm inhibiting peptides sequences and their features were used for the training.
	</tr>  
		  
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/proinflam.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-016-0928-3">
                <papertitle>ProInflam: a webserver for the prediction of proinflammatory antigenicity of peptides and proteins</papertitle> 
	      </a>
              <br>
	      <a href="https://www.linkedin.com/in/sudheer-gupta-91369580?originalSubdomain=in">Sudheer Gupta</a>,
	      <a href="https://in.linkedin.com/in/midhun-k-madhu-9a909286">Midhun K. Madhu Dhakan</a>,
              <strong>Ashok K. Sharma</strong>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a>
              <br>
              <br>
              <em>Journal of Translational Medicine </em>, 2016
              <br>
	      <a href="https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-016-0928-3">paper</a> /
	      <a href="http://metagenomics.iiserb.ac.in/proinflam/">web page </a>
              <p></p>
              <p>
                  ProInFlam is a unique tool for the computational identification of proinflammatory peptide antigen/candidates and provides leads for experimental validations
	</tr>
		  
         <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/hype.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-2753-8">
                <papertitle>Prediction of peptidoglycan hydrolases- a new class of antibacterial proteins</papertitle> 
	      </a>
              <br>
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://scholar.google.co.in/citations?user=ML7X29AAAAAJ&hl=en">Sanjiv Kumar</a>,
	      <a href="https://www.linkedin.com/in/harishk0201/">Harish K.</a>,
              <a href="https://scholar.google.com/citations?user=Gcp89foAAAAJ&hl=en">Darshan Dhakan</a>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a>
              <br>
              <br>
              <em>BMC Genomics </em>, 2016
              <br>
	      <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-2753-8">paper</a> /
	      <a href="http://metagenomics.iiserb.ac.in/hype/">web page </a>
              <p></p>
              <p>
                The present tool helps in the identification and classification of novel peptidoglycan hydrolases from complete genomic or metagenomic ORFs. To our knowledge, this is the only tool available for the prediction of peptidoglycan hydrolases from genomic and metagenomic data.
        </tr>
			      
        <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/MAG.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2016.00469/full">
                <papertitle>Reconstruction of Bacterial and Viral Genomes from Multiple Metagenomes</papertitle> 
	      </a>
              <br>
	      <a href="https://www.linkedin.com/in/ankit-gupta-82a632103/">Ankit Gupta</a>,
	      <a href="https://scholar.google.co.in/citations?user=ML7X29AAAAAJ&hl=en">Sanjiv Kumar</a>,
	      <a href="https://www.linkedin.com/in/vishnu-prasoodanan-p-k-ph-d-7a672072/?originalSubdomain=in">Vishnu Prasoodanan</a>,
	      <a href="https://www.linkedin.com/in/harishk0201/">Harish K.</a>,
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a>
              <br>
              <br>
              <em>Frontiers in Microbiology </em>, 2016
              <br>
	      <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2016.00469/full">paper</a>
              <p></p>
              <p>
		In this work, Binning-Assembly approach has been proposed and demonstrated for the reconstruction of bacterial and viral genomes from 72 human gut metagenomic datasets. 	
          </tr>
		
         <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/woods.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://www.sciencedirect.com/science/article/pii/S0888754315000543">
                <papertitle>Woods: A fast and accurate functional annotator and classifier of genomic and metagenomic sequences</papertitle> 
	      </a>
              <br>
	      <strong>Ashok K. Sharma</strong>,
	      <a href="https://www.linkedin.com/in/ankit-gupta-82a632103/">Ankit Gupta</a>,
	      <a href="https://scholar.google.co.in/citations?user=ML7X29AAAAAJ&hl=en">Sanjiv Kumar</a>,
	      <a href="https://scholar.google.com/citations?user=Gcp89foAAAAJ&hl=en">Darshan Dhakan</a>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a>
              <br>
              <br>
              <em>Genomics </em>, 2015
              <br>
	      <a href="https://www.sciencedirect.com/science/article/pii/S0888754315000543">paper</a> /
              <a href="http://metagenomics.iiserb.ac.in/woods/index.php">web page </a>
              <p></p>
              <p>
               Integrated approach for fast and accurate functional annotation of proteins in both genomic and metagenomic datasets. It is available as stand-alone tool as well as a publicly available web-server.	
	</tr>
		
		<tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/16s_classifier.jpg" alt="PontTuset" width="160" style="border-style: none">
            </td>
              <td width="75%" valign="middle">
	      <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0116106">
                <papertitle>The 16S Classifier: A Tool for Fast and Accurate Taxonomic Classification of 16S rRNA Hypervariable Regions in Metagenomic Datasets</papertitle> 
	      </a>
              <br>
	      <a href="https://www.researchgate.net/profile/Nikhil-Chaudhary-10">Nikhil Chaudhary</a>,
	      <strong>Ashok K. Sharma**</strong>,
              <a href="https://www.linkedin.com/in/piyush-agarwal-b16702239/?trk=public_profile_browsemap&originalSubdomain=sg">Piyush Agarwal</a>,
	      <a href="https://www.linkedin.com/in/ankit-gupta-82a632103/">Ankit Gupta</a>,
	      <a href="https://bio.iiserb.ac.in/faculty_profile.php?id=MTM=&lname=dmluZWV0a3M=">Vineet K. Sharma</a>
              <br>
              <br>
              <em>PLOS ONE </em>, 2015
              <br>
	      <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0116106">paper</a> /
              <a href="http://metagenomics.iiserb.ac.in/16Sclassifier/application.php">web page </a>
              <p></p>
              <p>
		'16S Classifier' is a Random Forest based tool which is developed to carry out fast, efficient and accurate taxonomic classification of 16S rRNA sequences. It has the unique ability to classify small Hypervariable Regions of 16S rRNA.	     
	</tr>	
         
	</tbody></table>

	      
	  <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:0px">
              <br>
              <p style="text-align:right;font-size:small;">
              Source code credit to <a href="https://jonbarron.info/" target="_blank">Dr. Jon Barron</a></p>
            </td>
          </tr>
        </tbody></table>
      
      </td>
    </tr>
  </table>
</body>

</html>
